maraboupy/MarabouNetworkONNX.py

```diff
@@ -217,8 +225,15 @@ def makeGraphEquations(self, nodeName, makeEquations):
             raise RuntimeError(err_msg)
 
         # Compute node's shape and create Marabou equations as needed
+        if self.thresholdReached:
+            return
         self.makeMarabouEquations(nodeName, makeEquations)
 
+        if self.maxNumberOfLinearEquations is not None:
+            if not self.thresholdReached and len(self.equList) > self.maxNumberOfLinearEquations:
+                if self.splitNetworkAtNode(nodeName, networkNamePostSplit='post_split.onnx'):
```

Comment:
I think one should probably use a tempfile name, otherwise two threads running in parallel would be interfering